### DISCUSSION TITLE
Please make consistent colors between `btn` and `btn-outline`

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [saadeghi/daisyui#1921] [Q&A] [answered]
With the new version 3, the default color for buttons is a light color, while the dark version can be applied with `btn-neutral` modifier.

There seems to be an inconsistency when using the outline version of the buttons, the default color is the dark `neutral`, while the light version is not available at all.

It would be useful when applying these classes programmatically to have a consistent setting.

I propose to have a default light color also of the outline version, and to have to use `btn-neutral` t get the dark version.

**Comment by saadeghi** [comment, marked as answer]
<img width="644" alt="g33" src="https://github.com/saadeghi/daisyui/assets/7342023/2369555e-f045-4511-91b6-451b90862e50"> 
<img width="619" alt="55" src="https://github.com/saadeghi/daisyui/assets/7342023/dc8fc702-680b-425c-980f-8efc67837e40">


But an outline button with that color wouldn't be visible at all.  
Unless we make an exception for the text color and change the text color to something else.  
Or it should be only allowed to use on a really distinguishable background like `bg-primary` (and even that only gives us a good accessible color if the primary color has a good contrast comparing to the base color)


Is there a specific scenario that you need this color for outline button? Because inaccessible color contrasts like these are the exact reason we're not making color variations programmatically.
